src/compressio/typesets.py

```diff
@@ -0,0 +1,14 @@
+from visions import String, Object, Integer, Float, Complex, Generic
+from visions.typesets import VisionsTypeset
+
+
+class DefaultCompressioTypeset(VisionsTypeset):
+    def __init__(self):
+        types = [Object, String, Integer, Float, Complex, Generic]
+        super().__init__(types)
+
+
+class SparseCompressioTypeset(VisionsTypeset):
```

Comment:
Currently this typeset is unused and identical to the default typeset. If it has added value, can we show that in the sparse notebook?

https://github.com/dylan-profiler/compressio/blob/master/examples/notebooks/Sparse%20Data.ipynb

Reply:
We can absolutely remove this - I only added the Typeset because you had previously created a Compressor for Sparse with a different set of types than the DefaultCompressor. I didn't put much effort into it but the SparseCompressor includes DateTime and Boolean. How do you want this to work?

Reply:
I've updated the typeset to accurately reflect the types used in the SparseCompressor. This PR has already grown out of scope so I will create a SparseCompress that uses the SparseTypeset and previous SparseCompressor.